ios/RNSScreenStack.mm

```diff
@@ -489,7 +569,12 @@ - (void)updateContainer
   NSMutableArray<UIViewController *> *pushControllers = [NSMutableArray new];
   NSMutableArray<UIViewController *> *modalControllers = [NSMutableArray new];
   for (RNSScreenView *screen in _reactSubviews) {
-    if (!screen.dismissed && screen.controller != nil) {
+#ifdef RN_FABRIC_ENABLED
+    if (screen.controller != nil)
+#else
+    if (!screen.dismissed && screen.controller != nil)
```

Comment:
Didn't we add `dismissed` in the previous PR? I think we can now remove this `ifdef` if so.